tests/unauthorized/filesTest.js

```diff
@@ -130,7 +139,7 @@ describe('Unauthorized: Currently testing files management,', function () {
   it('checking method : delete', function (done) {
     const newFolder = config.testFolder + '/' + 'new folder'
 
-    oc.files.mkdir(newFolder).then(() => {
+    oc.files.delete(newFolder).then(() => {
```

Comment:
The interaction is to delete a directory, so it makes more sense deleting a directory instead of making a directory